draft-labelling-describing.md

```diff
@@ -0,0 +1,105 @@
+Labeling and Describing
+
+## Labels
+
+A label is used as the accessible name for an element.
```

Comment:
we may not want to assume the term accessible name is meaningful to readers.

Reply:
Good point. Filed https://github.com/w3c/aria-practices/issues/996